### PR TITLE
Add a run-time saving/loading demo project

### PR DESCRIPTION
**Pull request by Calinou**
This is useful to load user-generated content without requiring users to create a PCK file for it.

**Marked as draft**, as this shouldn't be merged before 4.2 is released (this adds support for a feature not found in 4.1.x, namely run-time SVG loading).

- See https://github.com/godotengine/godot-docs/issues/6385,
  https://github.com/godotengine/godot/pull/78248 and
  https://github.com/godotengine/godot-proposals/issues/1632.

## Preview

image

### CODE REVIEW

File: loading/runtime_save_load/runtime_save_load.gd [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+		audio_player.visible = true
+
+	# 3D scenes.
+	elif path_lower.ends_with(".gltf") or path_lower.ends_with(".glb"):
+		# GLTFState is used by GLTFDocument to store the loaded scene's state.
+		# GLTFDocument is the class that handles actually loading glTF data into a Godot node tree,
+		# which means it supports glTF features such as lights and cameras.
+		var gltf_document := GLTFDocument.new()
+		var gltf_state := GLTFState.new()
+		var error := gltf_document.append_from_file(path, gltf_state)
```
Comment (fire) [inline loading/runtime_save_load/runtime_save_load.gd:185]: Missing base path. @Calinou

Comment (Calinou) [reply loading/runtime_save_load/runtime_save_load.gd:184]: Base path should be set automatically according to this code:

https://github.com/godotengine/godot/blob/5ee983188de97ae027f9b9c1443438063f708a7e/modules/gltf/gltf_document.cpp#L7627-L7630

Comment (fire) [reply loading/runtime_save_load/runtime_save_load.gd:185]: you can load glbs as buffers and there is no path. But this allows you to relocate the path to extra buffers and images for gltfs.

Comment (Calinou) [reply loading/runtime_save_load/runtime_save_load.gd:185]: I see, I'll open a PR to the documentation to mention ``base_path`` there.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**